pkgs/applications/networking/cluster/kubectl-ktop/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ktop";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "vladimirvivien";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-9TKXOKGsxtmyTGGVq7GDy4v3I9BF5SDc48dhrj1pw9Q";
+  };
+
+  vendorSha256 = "sha256-IbWdq6t9tgly7MRRIRJK5sDHus+sF0AT843aI4quO3c=";
+
+  postConfigure = ''
+    rm -rf ./.ci ./hack
+  '';
```

I guess you added this because it otherwise tries to build those sub packages?
Use the following instead:

```suggestion
  subPackages = [ "." ];
```